covid_graphs/covid_graphs/prediction_generator.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+from pathlib import Path
+from typing import List
+
+import click
+import click_pathlib
+from google.protobuf import text_format  # type: ignore
+
+from .country_report import CountryReport, create_report
+from .formula import FittedFormula
+from .pb.atg_prediction_pb2 import AtgTraceGenerator, AtgTraceGenerators
+from .predictions import CountryPrediction, PredictionEvent
+
+PREDICTION_DAYS = 21
+
+
+def get_fitted_predictions(
+    report: CountryReport, dates: List[datetime.date]
+) -> List[CountryPrediction]:
+    return list(
+        CountryPrediction(
+            prediction_event=PredictionEvent(
+                name=f"daily_fit_{last_data_date.strftime('%Y_%m_%d')}",
+                last_data_date=last_data_date,
+                prediction_date=last_data_date,
+            ),
+            country=report.short_name,
+            formula=FittedFormula(until_date=last_data_date, country_report=report),
+        )
+        for last_data_date in dates
```

Comment:
Can we create non-sensical predictions here? Maybe we should add:
```py
if last_data_date in report.dates
```

Reply:
Done